### DISCUSSION TITLE
admin_CoEHttpAzureAD

### DISCUSSION THREAD

**Discussion by seschn** [microsoft/powerapps-tools#1012] [Q&A]
Trying to install the May 2021 update for the Dataverse Core Components and it's asking me for the Base Resource URL and Azure AD Resource URI for a new connection called admin_CoEHttpAzureAD. Can't proceed with the install without providing a connection for this component.

Any background info on what this item does and how I should set it up?

**Comment by manuelap-msft** [comment]
This is covered in the setup instructions https://docs.microsoft.com/en-us/power-platform/guidance/coe/setup-core-components#option-1-import-the-solution-into-a-production-environment

![image](https://user-images.githubusercontent.com/55705669/117937345-c8c8b980-b2fd-11eb-9a62-f1dd0972fe3c.png)

We use the HTTP with Azure AD connector to get information on Service Principals - e.g. if your flow or app is owned by a Service Principal, we use this connector to find out the SPN name and list the Owner appropriately.

**Reply by shyjujohn** [reply]
Hi @manuelap-msft 
What are the permissions assigned to this connector, Read or Read/write? Any additional information available on this Connector.

**Reply by manuelap-msft** [reply]
This repo is no longer used or monitored by the CoE starter kit team. The new repo where you can raise issues is https://github.com/microsoft/coe-starter-kit/issues/new/choose 

We have updated all our docs to point to the new repo; please update your bookmarks if you still have the old link. Thank you.
